monitoring_service/state_db/sqlite_db.py

```diff
@@ -9,16 +9,30 @@
 from raiden_libs.types import Address, ChannelIdentifier
 from raiden_libs.utils import is_channel_identifier
 
-from .queries import ADD_MONITOR_REQUEST_SQL, DB_CREATION_SQL, UPDATE_METADATA_SQL
+
+SCHEMA_FILENAME = os.path.join(
+    os.path.dirname(os.path.realpath(__file__)),
+    'schema.sql',
+)
+
+
+def convert_hex(raw: bytes):
+    return int(raw, 16)
+
+
+sqlite3.register_converter('HEX_INT', convert_hex)
```

Comment:
This only seems to do the conversion back to int when querying, can we do this automatically for storing as well?